shared/components/modal/ModalConductor/ModalConductor.js

```diff
@@ -35,9 +33,9 @@ export default class ModalConductor extends Component {
       document.body.style.paddingRight = `${getScrollBarWidth()}px`
 
       if (myOffsetTop > 0) {
-        this.setState({
+        return {
           offsetTop: myOffsetTop,
-        })
+        };
```

Comment:
```suggestion
        }
```

мы вроде не ставим точки с запятой по проекту, мб лучше убрать